packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -343,12 +385,23 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
       let item = renderedItems[i];
       let itemWidth = this._commandItemWidths![item.key];
 
+      if (!itemWidth) {
```

Comment:
Why would the value be null?

Reply:
If the user passes an unsupported ItemType (such as section or header) to the contextmenu, then the stored itemwidth will be null.